packages/pluggableWidgets/accordion-web/src/Accordion.editorConfig.ts

```diff
@@ -10,8 +17,35 @@ export function getProperties(
     values.groups.forEach((group, index) => {
         if (group.headerRenderMode === "text") {
             hidePropertyIn(defaultProperties, values, "groups", index, "headerContent");
+
+            if (!values.advancedMode) {
+                hidePropertyIn(defaultProperties, values, "groups", index, "headerHeading");
+            }
         } else {
             hidePropertyIn(defaultProperties, values, "groups", index, "headerText");
+            hidePropertyIn(defaultProperties, values, "groups", index, "headerHeading");
+        }
+
+        if (group.initialCollapsedState !== "dynamic") {
+            hidePropertyIn(defaultProperties, values, "groups", index, "initiallyCollapsed");
+        }
+
+        if (!values.advancedMode) {
+            hideNestedPropertiesIn(defaultProperties, values, "groups", index, [
+                "collapsed",
+                "onToggleCollapsed",
+                "initialCollapsedState",
+                "initiallyCollapsed"
+            ]);
+        }
+
+        if (!values.collapsible) {
+            hideNestedPropertiesIn(defaultProperties, values, "groups", index, [
+                "collapsed",
+                "onToggleCollapsed",
+                "initialCollapsedState",
+                "initiallyCollapsed"
+            ]);
         }
```

Comment:
These two conditionals can be combined right? It looks like they're hiding the same properties

Reply:
You are right! Changed it.